selfdrive/car/hyundai/values.py

```diff
@@ -1519,6 +1520,7 @@ class Buttons:
       b'\xf1\0006U3L0_C2\000\0006U3K3051\000\000HCN0G16NS0\000\000\000\000',
       b'\xf1\x816U3K3051\000\000\xf1\0006U3L0_C2\000\0006U3K3051\000\000HCN0G16NS0\xb9?A\xaa',
       b'\xf1\x816U3K3051\x00\x00\xf1\x006U3L0_C2\x00\x006U3K3051\x00\x00HCN0G16NS0\x00\x00\x00\x00',
+      b'\xf1\x816U3K9051\x00\x00\xf1\x006U3L0_C2\x00\x006U3K9051\x00\x00HCN0G16NS1\x9f\x93\xa8\xf7',
```

Comment:
In [your carFw](https://useradmin.comma.ai/logs?url=https%3A%2F%2Fcommadata2.blob.core.windows.net%2Fqlog%2F9b1bd7e1e54c6a37%2F2023-04-26--08-00-24%2F0%2Fqlog.bz2%3Fse%3D2023-04-27T01%253A16%253A48Z%26sp%3Dr%26sv%3D2018-03-28%26sr%3Db%26rscd%3Dattachment%253B%2520filename%253D9b1bd7e1e54c6a37_2023-04-26--08-00-24--0--qlog.bz2%26sig%3DE%2Fixbyn6xoX8Jbwbe35rb4TewK3cGT75YMjcL6e%252Byys%253D&event=carParams), the last 4 bytes of this FW are different:
```
b'\xf1\x816U3K9051\x00\x00\xf1\x006U3L0_C2\x00\x006U3K9051\x00\x00HCN0G16NS1\x00\x00\x00\x00'
```

Where did you copy this version from originally?

Reply:
Since my device ID was banned, I used the method outlined here: https://github.com/commaai/openpilot/wiki/Fingerprinting#fingerprinting-20-alternative-for-banned-non-official-or-non-commaai-devices
Sorry, I did not save the terminal log. If you can get what is needed now from my uploaded drive segments now that the device ID is unbanned, perhaps that is the preferred source?